internal/livetiming/livetiming.go

```diff
@@ -20,7 +22,12 @@ func Init() error {
 		if err != nil {
 			return
 		}
-		defer f.Close()
+		defer func(f *os.File) {
+			err := f.Close()
+			if err != nil {
+				log.Println("issue closing paths.json file: ", err)
```

Comment:
This is a bit nitpicky, but this is another example of something that could be squashed into the commit which introduced the deferred `f.Close`. Just to avoid confusion and to keep this commit to one logical change